src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -231,42 +319,53 @@ export default class PhotoField extends React.Component {
     this.setState({ dragAndDropSupported: dragAndDropSupported && !iOS });
   }
 
-  detectWidth = () => {
-    const windowWidth = window.innerWidth;
-    this.setState({ windowWidth });
-  }
-
-  moveUp = () => {
-    this.refs.cropper.move(0, -5);
-  }
+  updateBoundaryWarningAndButtonStates = () => {
+    const moveBoundariesMet = getMoveBoundariesMet(this.refs.cropper.getCanvasData(), this.refs.cropper.getCropBoxData());
 
-  moveDown = () => {
-    this.refs.cropper.move(0, 5);
+    this.setState({ ...makeMoveButtonsEnabledStates(moveBoundariesMet) });
+    this.setState({
+      warningMessage: makeCropBoundaryWarningMessage(moveBoundariesMet),
+      ...makeMoveButtonsEnabledStates(moveBoundariesMet)
+    });
   }
 
-  moveRight = () => {
-    this.refs.cropper.move(-5, 0);
+  cropend = () => { // casing matches Cropper argument
```

Comment:
I think maybe `onCropend` or `handleCropend` would be better, since it looks like we've been using those two conventions for event handler methods.